CHANGELOG.md

```diff
@@ -2,9 +2,44 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [1.22.0](https://github.com/aws/jsii/compare/v1.21.0...v1.22.0) (2021-02-18)
+## [1.23.0](https://github.com/aws/jsii/compare/v1.22.0...v1.23.0) (2021-03-01)
+
+
+### ⚠ BREAKING CHANGES
+
+* **java:** in TypeScript, this allows us to preserve this property
+in Java, too).
+
+However, if two distinct super-interfaces declare the same property, the
+child inherits two unrelated default implementations for the same
+method, which results in dispatch ambiguity that is illegal.
+
+In order to remove the ambiguity, a new default implementation must be
+provided on the child interface, so that the dispatch is unambiguous
+again.
+
```

Comment:
This whole section was incorrectly picked up from https://github.com/aws/jsii/commit/23996087adef48f73ecb61d6b499263435af2283 due to how the commit message was flowed (the sequence `breaking change` appears at the beginning of a line, but is actually mid-sentence at this stage).

```suggestion
```